### DISCUSSION TITLE
is this dead?

### DISCUSSION THREAD

**Discussion by UnknownSkid** [stakira/OpenUtau#1145] [General]
is OpenUtau Dead? is there gonna be updates

**Comment by Nexys4t** [comment]
OpenUTAU gets updated very frequently. If you think it's dead, you didn't even look neither the releases page or the main page.

**Reply by UnknownSkid** [reply]
theyre all pre-releases

**Reply by Nexys4t** [reply]
Yeah, that's the point. It's a program in development so it's normal that most of the contents are pre-releases.

It's disrespectful to ask if a program is dead when it's getting an active development.

**Reply by UnknownSkid** [reply]
im sorry

**Reply by UnknownSkid** [reply]
does the pre-release work correctly, can i download it and it wont crash

**Comment by Cardroid** [comment]
It is a feature of OSS.
If you have the features you need or need additional updates, you can implement them yourself and submit PR. 👍
